streaming/redpanda/main.tf

```diff
@@ -0,0 +1,147 @@
+################################################################################
+# Data 
+################################################################################
+
+data "aws_availability_zones" "available" {}
+data "aws_caller_identity" "current" {}
+data "aws_partition" "current" {}
+
+data "aws_secretsmanager_secret_version" "redpanada_password_version" {
+    secret_id = aws_secretsmanager_secret.redpanada_password.id
+    depends_on = [aws_secretsmanager_secret_version.redpanada_password]
+}
+data "aws_secretsmanager_secret_version" "grafana_password_version" {
+    secret_id = aws_secretsmanager_secret.redpanada_password.id
+    depends_on = [aws_secretsmanager_secret_version.grafana_password_version]
+}
+
+################################################################################
+# Local Variables
+################################################################################
+locals {
+    name = var.name
+    region = var.region
+    account_id = data.aws_caller_identity.current.account_id
+    partition = data.aws_partition.current.partition
+  
+    vpc_cidr = var.vpc_cidr
+    azs = slice(data.aws_availability_zones.available.names, 0 ,2)
+
+
+    tags = {
+        
+    }
+
+}
+
+#---------------------------------------
```

Comment:
Please move Redpanda Config, Grafana Admin credentials in addons.tf

Reply:
complete